### ISSUE TITLE
Marking all projects in dir as protected

### ISSUE THREAD

**Issue by vrujbr** [rockbruno/swiftshield#35]
In last part of the run, xcodeproj is marked as SWIFTSHIELDED.
Find findFiles method finds all the xcodeproj files in the directory, which might be unwanted behaviour (at least for me it is).

**Comment by rockbruno** [comment]
I think that ended up being a leftover from the manual mode, where everything is marked because no specific project file is mentioned

I was thinking if it was possible for someone to add some arbitrary script to a Pods project, but I assume nobody who needs to do that has an use for this tool, so that's definitely something that could be improved 🙂 

The only discussion point needed is that you can link an workspace, so we don't really know which project is the "right" one (and you can name your workspace something else, in theory), so I wonder how we could determine which project to mark as protected...

**Comment by vrujbr** [comment]
Yep agreed. Well you could either disable it for manual mode of force the user to specify project file even in manual..
Anyway its not a big deal, it was for me as I had one old leftover empty xcodeproj file and it was crashing the swiftshiled at the end.

**Comment by rockbruno** [comment]
Do you remember why it was crashing? That's really bad
Specifying an array of project files sounds good to me actually, I'm not a fan of mindlessly scanning files like it currently does

**Comment by vrujbr** [comment]
Ok I wasn't quite correct, it wasn't crashing, it was exiting the app with non 0 exit code.
In `markProjectsAsProtected` it checks path `projectPath+"/project.pbxproj"`, throws exception, catches and exits.
Steps to reproduce: add file/dir with .xcodeproj extension to your project folder.
